src/rgw/support/rgw_support_bundle

```diff
@@ -164,21 +243,100 @@ class RGWSupportBundle:
 
         # copy ceph crash-dump files
         if coredumps:
-            crash_dump_dir = os.path.join(log_dir, RGW_CORE_FILE_DIR_NAME)
+            crash_dump_dir = os.path.join(log_dir, 'rgw_debug')
+            dest_dump_dir = os.path.join(RGWSupportBundle._tmp_src, 'rgw_debug')
             if os.path.exists(crash_dump_dir):
-                shutil.copytree(crash_dump_dir,\
-                    os.path.join(RGWSupportBundle._tmp_src, RGW_CORE_FILE_DIR_NAME))
+                if size_limit:
+                    # get latest 2 core dumps
+                    latest_cores = RGWSupportBundle._get_latest_files(
+                        directory=crash_dump_dir, number_of_files=2)
+                    if len(latest_cores) == 2:
+                        remaining_size = RGWSupportBundle._get_remaining_folder_size(
+                            RGWSupportBundle._tmp_src, size_limit)
+                        # check if sum of both cores is less than remaining size else collect only one
+                        sum_of_core_size = os.path.getsize(
+                            os.path.join(crash_dump_dir,latest_cores[0])) + \
+                            os.path.getsize(
+                                os.path.join(crash_dump_dir,latest_cores[1]))
+                        # select only one core if both cores exceed the size limit
+                        if sum_of_core_size > int(remaining_size[:-1]):
+                            latest_cores.pop()
```

Comment:
this will pop second to the last core right, so latest core file will still be collected

Reply:
yes.